expression/builtin_time_vec_test.go

```diff
@@ -190,6 +190,9 @@ var vecBuiltinTimeCases = map[string][]vecExprBenchCase{
 			geners: []dataGenerator{gener{defaultGener{eType: types.ETDecimal, nullRation: 0.9}}},
 		},
 	},
+	ast.StrToDate: {
+		{retEvalType: types.ETDatetime, childrenTypes: []types.EvalType{types.ETString, types.ETString}},
```

Comment:
It'd be better implement a new test generator instead of the default random string generator.

Reply:
Thanks. Fixed.